
Loaded checkpoint gp_mobilenet_v2_DH0.9_DL1.0_W0.35_local_1x2_T at epoch 2.
Traceback (most recent call last):
  File "train.py", line 637, in <module>
    main()
  File "train.py", line 635, in main
    train_val_test()
  File "train.py", line 616, in train_val_test
    cal_meters, phase='cal',
NameError: name 'cal_meters' is not defined
Traceback (most recent call last):
  File "train.py", line 637, in <module>
    main()
  File "train.py", line 635, in main
    train_val_test()
  File "train.py", line 616, in train_val_test
    cal_meters, phase='cal',
NameError: name 'cal_meters' is not defined
Files already downloaded and verified
Files already downloaded and verified
Start group-level pruning version training.
Start validation after calibration.